YouVsVirus/Assets/Scenes/Levelgethome/YouVsVirus_Levelgethome.unity

```diff
@@ -868,9 +868,9 @@ Camera:
   m_NormalizedViewPortRect:
     serializedVersion: 2
     x: 0
-    y: 0
+    y: 0.00031727552
     width: 1
-    height: 1
+    height: 0.99936545
```

Comment:
Would love to understand why this happens.

You change  nothing in unity but some magic number in the scene switches from 1 to 0.9993.

Reply:
yes, i find that strange, too.